examples/extensions/r2/deflat_r2.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+#
+# Cross Platform and Multi Architecture Advanced Binary Emulation Framework
+#
+
+import sys
+
+sys.path.append('..')
+
+from qiling import Qiling
+from qiling.const import QL_VERBOSE
+from qiling.extensions.r2 import R2Qiling
+
+
+
+if __name__ == "__main__":
+    # a program obfuscated by OLLVM CFF flatten, which should print 4 when argv[1] is 1
+    # see source code at examples/src/linux/fla_test.c
+    ql = R2Qiling(['rootfs/x86_linux/bin/test_fla_argv', '1'], 'rootfs/x86_linux', verbose=QL_VERBOSE.DEFAULT)
```

Comment:
I think that removing the Qiling import and importing r2q as Qiling would be a cleaner approach; e.g.
```python
from qiling.extensions.r2 import R2Qiling as Qiling
# ...
ql = Qiling(...)
```